exchanges/execute/src/execute.ts

```diff
@@ -24,7 +24,7 @@ import {
   getOperationName,
 } from '@urql/core';
 
-interface ExecuteExchangeArgs {
+export interface ExecuteExchangeArgs {
```

Comment:
This will also need exporting from `./src/index.ts`

Reply:
oops I messed up... updated!